Source/WebCore/PAL/Configurations/PAL.xcconfig

```diff
@@ -110,3 +110,5 @@ EXCLUDED_SOURCE_FILE_NAMES[sdk=watchos10*] = ;
 EXCLUDED_SOURCE_FILE_NAMES[sdk=watchsimulator10*] = ;
 EXCLUDED_SOURCE_FILE_NAMES[sdk=xros1*] = ;
 EXCLUDED_SOURCE_FILE_NAMES[sdk=xrsimulator1*] = ;
+
+EXCLUDED_ARCHS = i386;
```

Comment:
It might be nice to more explicitly target this at watchOS Simulator:

```suggestion
EXCLUDED_ARCHS = $(EXCLUDED_ARCHS_$(WK_PLATFORM_NAME));
EXCLUDED_ARCHS_watchsimulator = i386;
```

Also, a comment mentioning that this is behavior in the watchOS 10 SDK would be helpful for indicating when we may be able to clean this up.